modules/persistent-storage-csi-vsphere-top-aware.adoc

```diff
@@ -105,14 +163,14 @@ co8-4s88d-worker-zlk7d      1           47m
 ----
 ...
 spec:
-    drivers:
-    - allocatable:
-        count: 59
-    name: csi-vsphere.vmware.com
-    nodeID: co8-4s88d-worker-j2hmg
-    topologyKeys: <1>
-    - topology.csi.vmware.com/openshift-zone
-    - topology.csi.vmware.com/openshift-region
+drivers:
+- allocatable:
+count: 59
+name: csi-vsphere.vmware.com
+nodeID: co8-4s88d-worker-j2hmg
+topologyKeys: <1>
+- topology.csi.vmware.com/openshift-zone
+- topology.csi.vmware.com/openshift-region
```

Comment:
Looks like the format is not correct? I think the previous is correct one.

Reply:
Fixed.